src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -889,14 +887,18 @@ private async Task<(long, object)> ModifyTimeSeriesConfiguration(TransactionOper
             var editTimeSeries = new EditTimeSeriesConfigurationCommand(configuration, name, raftRequestId);
             var result = await ServerStore.SendToLeaderAsync(editTimeSeries);
 
-            DatabaseTopology topology;
-            ClusterTopology clusterTopology;
+            List<string> members;
             using (context.OpenReadTransaction())
+                members = ServerStore.Cluster.ReadDatabaseTopology(context, name).Members;
+
+            try
+            {
+                await ServerStore.WaitForExecutionOnRelevantNodesAsync(context, members, result.Index);
+            }
+            catch (Exception e) when (e is AggregateException)
```

Comment:
would those exceptions cause the client to failover? do we want it to failover?

Reply:
As far as I can see, no, those exceptions will not cause the client to failover. And I see no reason for triggering the failover mechanism.